actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -14,7 +14,17 @@ class ERB
         class_attribute :erb_implementation, default: Erubi
 
         # Do not escape templates of these mime types.
-        class_attribute :escape_whitelist, default: ["text/plain"]
+        class_attribute :escape_permit_list, default: ["text/plain"]
```

Comment:
"permit" doesn't really apply here. I was trying to come up with a better overall phrasing, but "non escaped types" sounds awful... `escape_exclude_list` would at least imply the relationship is the right way around.

Reply:
I tried `escape_ignore_list` because it seems like that's more what it's doing